undo_telemetry.sh

```diff
@@ -25,20 +26,20 @@ else
 fi
 export -f replace_with_debug
 
-d1=`date +%s`
+d1=$(date +%s)
 
 if [[ "${OS_NAME}" == "linux" ]]; then
   if [[ ${VSCODE_ARCH} == "x64" ]]; then
-    ./node_modules/@vscode/ripgrep/bin/rg --no-ignore -l "${SEARCH}" . | xargs -I {} bash -c 'replace_with_debug "${1}" "{}"' _ "${REPLACEMENT}"
+    ./node_modules/@vscode/ripgrep/bin/rg --no-ignore -l "${SEARCH}" . | xargs -I {} bash -c "replace_with_debug \"${1}\" \"{}\"" _ "${REPLACEMENT}"
```

Comment:
The single quote are ok here

Reply:
I will change these back and ignore the message. I wasn't sure if there could be an edge case where the expansion of `"${1}"` might be an issue, but I don't think so.